packages/generate-pdf/example_coa.ts

```diff
@@ -51,29 +52,19 @@ const generatorPath = '../generate-coa-pdf-template/dist/generateContent.cjs';
 
     const pdfDoc = await generatePdf(CoACertificate, {
       docDefinition,
-      outputType: 'stream',
+      outputType: 'buffer',
       generatorPath,
       fonts,
       extraTranslations,
       translations,
       languageFontMap,
+      attachCertificate: true,
     });
 
     const outputFilePath = './coa-test.pdf';
-    const writeStream = createWriteStream(outputFilePath);
-    pdfDoc.pipe(writeStream);
-    pdfDoc.end();
-
-    await new Promise((resolve, reject) => {
-      writeStream
-        .on('finish', () => {
-          resolve(true);
-        })
-        .on('error', (err) => {
-          reject(err);
-        });
-    });
+    await writeFile(outputFilePath, pdfDoc);
```

Comment:
Maybe now is a good time to start measuring how this example performs?
For example using [`perf_hooks`](https://nodejs.org/api/perf_hooks.html#perf_hooksperformance) to check durations and `process. memoryUsage()` to track memory consumption (or even better with the heap profiler).

Reply:
In another PR we can do that. This one is too large already.